include/RandLAPACK/comps/util.hh

```diff
@@ -0,0 +1,134 @@
+#ifndef BLAS_HH
+#include <blas.hh>
+#define BLAS_HH
+#endif
+
+namespace RandLAPACK::comps::util {
+
+template <typename T>
+void eye(
+        int64_t m,
+        int64_t n,
+        std::vector<T>& A
+);
+
+template <typename T> 
+void get_L(
+        int64_t m,
+        int64_t n,
+        std::vector<T>& L
+);
+
+template <typename T>
+void diag(
+        int64_t m,
+        int64_t n,
+        const std::vector<T>& s, // pointer to the beginning
+        int64_t k,
+        std::vector<T>& S
+);
+
+template <typename T> 
+void disp_diag(
+        int64_t m,
+        int64_t n,
+        int64_t k, 
+        std::vector<T>& A 
+);
```

Comment:
Is ``k`` the number of elements to display, or the offset of the diagonal (e.g., ``k=1`` means first superdiagonal, ``k=-2`` means second subdiagonal)?